specification/frontdoor/resource-manager/Microsoft.Network/stable/2020-01-01/frontdoor.json

```diff
@@ -1570,14 +1570,21 @@
           "description": "Treatment of URL query terms when forming the cache key.",
           "enum": [
             "StripNone",
-            "StripAll"
+            "StripAll",
+            "StripOnly",
+            "StripAllExcept"
           ],
           "type": "string",
           "x-ms-enum": {
             "name": "FrontDoorQuery",
             "modelAsString": true
           }
         },
+        "queryParameters": {
+          "description": "query parameters to include or exclude (comma separated).",
+          "type": "string",
+          "x-nullable": true
```

Comment:
>x-nullable": true [](start = 11, length = 17)

do we need this?

seems like even recently there's problem with x-nullable in autorest 
[https://github.com/Azure/autorest/issues/3300](https://github.com/Azure/autorest/issues/3300)

Reply:
Thanks for letting me know, I removed x-nullable field.

---
In reply to: [382354208](https://github.com/Azure/azure-rest-api-specs/pull/8455#discussion_r382354208) [](ancestors = 382354208)